acme_tiny.py

```diff
@@ -48,7 +48,7 @@ def _do_request(url, data=None, err_msg="Error", depth=0):
 
     # helper function - make signed requests
     def _send_signed_request(url, payload, err_msg, depth=0):
-        payload64 = _b64(json.dumps(payload).encode('utf8'))
+        payload64 = "" if payload is None else _b64(json.dumps(payload).encode('utf8'))
         new_nonce = _do_request(directory['newNonce'])[2]['Replay-Nonce']
```

Comment:
You need to do a POST-as-GET here as well.

Reply:
Hmmm, I thought `newNonce` must continue to accept `GET` requests?

https://tools.ietf.org/html/rfc8555#section-7.2

Reply:
Ah yes, sure. I ended up putting this comment in the completely wrong line :)